cli/internal/fs/path.go

```diff
@@ -151,6 +162,19 @@ func IofsRelativePath(fsysRoot string, absolutePath string) (string, error) {
 	return filepath.Rel(fsysRoot, absolutePath)
 }
 
+// GetTempDir returns the absolute path of a directory with the given name
+// under the system's default temp directory location
+func GetTempDir(subDir string) AbsolutePath {
```

Comment:
Maybe just `TempDir()` to match the other method name?

Reply:
Done.